pkgs/development/python-modules/opentsne/default.nix

```
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, cython
+, numpy
+, scipy
+, scikit-learn
+}:
+
+buildPythonPackage rec {
+  pname = "openTSNE";
```

```suggestion
  pname = "opentsne";
```
> Attribute names in python-packages.nix as well as pnames should match the library's name on PyPI, but be normalized according to [PEP 0503](https://www.python.org/dev/peps/pep-0503/#normalized-names).

https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-guidelines-contributing-guidelines